docs/source/build-run-openvino.md

```diff
@@ -76,92 +78,33 @@ Follow the steps below to setup your build environment:
+cd executorch/examples/openvino
+python aot_optimize_and_infer.py --export --suite torchvision --model resnet50 --input_shape "(1, 3, 256, 256)" --device CPU
 ```
 The exported model will be saved as 'resnet50.pte' in the current directory.
 
-#### **Arguments**
-- **`--suite`** (required):  
-  Specifies the model suite to use.  
-  Supported values:
-  - `timm` (e.g., VGG16, ResNet50)
-  - `torchvision` (e.g., resnet18, mobilenet_v2)
-  - `huggingface` (e.g., bert-base-uncased)
-
-- **`--model`** (required):  
-  Name of the model to export.  
-  Examples:
-  - For `timm`: `vgg16`, `resnet50`
-  - For `torchvision`: `resnet18`, `mobilenet_v2`
-  - For `huggingface`: `bert-base-uncased`, `distilbert-base-uncased`
-
-- **`--input_shape`** (required):  
-  Input shape for the model. Provide this as a **list** or **tuple**.  
-  Examples:
-  - `[1, 3, 224, 224]` (Zsh users: wrap in quotes)
-  - `(1, 3, 224, 224)`
-
-- **`--device`** (optional):  
-  Target device for the compiled model. Default is `CPU`.  
-  Examples: `CPU`, `GPU`
-
 ### Build C++ OpenVINO Examples
-Build the backend and the examples by executing the script:
-```bash
-./openvino_build_example.sh
-```
-The executable is saved in `<executorch_root>/cmake-openvino-out/examples/openvino/`
-
-Now, run the example using the executable generated in the above step. The executable requires a model file (`.pte` file generated in the aot step), number of inference iterations, and optional input/output paths.
-
-#### Command Syntax:
-
-```
-cd ../../cmake-openvino-out/examples/openvino
 
-./openvino_executor_runner \
-    --model_path=<path_to_model> \
-    --num_iter=<iterations> \
-    [--input_list_path=<path_to_input_list>] \
-    [--output_folder_path=<path_to_output_folder>]
-```
-#### Command-Line Arguments
+After building the OpenVINO backend following the [instructions](#setup) above, the executable will be saved in `<executorch_root>/cmake-out/backends/openvino/`.
 
-- `--model_path`: (Required) Path to the model serialized in `.pte` format.
-- `--num_iter`: (Optional) Number of times to run inference (default: 1).
-- `--input_list_path`: (Optional) Path to a file containing the list of raw input tensor files.
-- `--output_folder_path`: (Optional) Path to a folder where output tensor files will be saved.
+The executable requires a model file (`.pte` file generated in the aot step) and the number of inference executions.
 
 #### Example Usage
 
-Run inference with a given model for 10 iterations and save outputs:
-
-```
-./openvino_executor_runner \
-    --model_path=model.pte \
-    --num_iter=10 \
-    --output_folder_path=outputs/
-```
-
-Run inference with an input tensor file:
+Run inference with a given model for 10 executions:
 
 ```
 ./openvino_executor_runner \
     --model_path=model.pte \
-    --num_iter=5 \
-    --input_list_path=input_list.txt \
-    --output_folder_path=outputs/
+    --num_executions=10
 ```
 
-## Supported model list
 
-### TODO
 
 ## FAQ
```

Comment:
This is not a new addition, but FAQ doesn't seem right for the below note. Let me know your thoughts

Reply:
Makes sense, I changed the header to "Support".